### ISSUE TITLE
logging: misleading query times reported

### ISSUE THREAD

**Issue by bwalsh** [bmeg/grip#142]
For example, this query:

```
{'query': [{'v': []}, {'where': {'condition': {'key': '_label', 'value': 'Biosample', 'condition': 'EQ'}}}, {'out': []}, {'where': {'condition': {'key': '_label', 'value': 'Individual', 'condition': 'EQ'}}}, {'count': ''}], 'base_url': 'http://arachne.compbio.ohsu.edu', 'url': 'http://arachne.compbio.ohsu.edu/v1/graph/bmeg-test/query', 'graph': 'bmeg-test', 'user': None, 'password': None}
```

produces this log message

```
[INFO]	2018-09-13 20:30:18,152	total: 1 results processed in 0 seconds
```

In reality, it took 21.77 seconds.

**Comment by adamstruck** [comment]
Those logs reflect client side processing. The rate reported has nothing to do with server-side query time. 

I can see how this would be confusing though.

**Comment by kellrott** [comment]
Fixed in #146 ?
